source/common/common/utility.cc

```diff
@@ -435,7 +435,7 @@ std::string StringUtil::subspan(absl::string_view source, size_t start, size_t e
   return std::string(source.data() + start, end - start);
 }
 
-std::string StringUtil::escape(const std::string& source) {
+std::string StringUtil::escape(absl::string_view source) {
```

Comment:
To preserve original constness, please consider.
```suggestion
std::string StringUtil::escape(const absl::string_view source) {
```

Reply:
sure